spyder/utils/icon_manager.py

```diff
@@ -198,6 +198,8 @@ def __init__(self):
             'rename':                  [('mdi.rename-box',), {'color': self.MAIN_FG_COLOR}],
             'move':                    [('mdi.file-move',), {'color': self.MAIN_FG_COLOR}],
             'edit_add':                [('mdi.plus',), {'color': self.MAIN_FG_COLOR}],
+            'collapcol':               [('mdi.arrow-collapse-horizontal',), {'color': self.MAIN_FG_COLOR}],
+            'collaprow':               [('mdi.arrow-collapse-vertical',), {'color': self.MAIN_FG_COLOR}],
```

Comment:
Let's change `collapcol` for `collapse_column` and `collaprow` for `collapse_row`. That's because I think it's better to be explicit than implicit and a few more characters won't hurt us.

Reply:
I agree!